### PR TITLE
frontend/fix/#2164-yatri-bug-fixes

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

File: Frontend/ui-customer/spago.dhall [hunk 1]
```diff
@@ -30,6 +30,7 @@ to generate this file without the comments in this block.
   , "functions"
   , "halogen-vdom"
   , "integers"
+  , "lists"
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/spago.dhall:33]: @MercyQueen Why is this required, I don't see any usage in this PR

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

**Comment by kavya-shree-s** [comment]
Can you squash the commit

File: Frontend/ui-customer/spago.dhall (hunk 1 above)
Comment (MercyQueen) [reply Frontend/ui-customer/spago.dhall:33]: @kavya-shree-s its not being used but while generating bundle, we are getting error, hence I've added it here.

**Review by kavya-shree-s — APPROVED** [review]
